src/plugins/table_vis/public/__tests__/_table_vis_controller.js

```diff
@@ -103,6 +107,23 @@ describe('Controller', function () {
     expect(!$scope.tableGroups).to.be.ok();
   });
 
+  it('sets the sort on the scope when it is passed as a vis param', function () {
+    const sortObj = {
+      columnIndex: 1,
+      direction: 'asc'
+    };
+    initController(new OneRangeVis({sort: sortObj}));
+
+    // modify the data to not have any buckets
+    var resp = _.cloneDeep(fixtures.oneRangeBucket);
```

Comment:
Scratch that.  This file is already inconsistent so it doesn't really matter.  My bulk `var->let` PRs will take care of this.